scripts/cover.sh

```diff
@@ -40,6 +40,7 @@ make munit-opae-c munit-opae-c-events
 
 lcov --directory . --zerocounters
 lcov -c -i -d . -o coverage.base
-./bin/munit-opae-c --gtest_filter="enum*:buffer*:*properties*:sysfs*"
+./bin/munit-opae-c --gtest_filter="mmio*"
```

Comment:
only mmio?

Reply:
I think this was done because he was testing only mmio. I don't think this script is final. It was meant as something to get up and running with coverage quickly.